twistes/consts.py

```diff
@@ -37,5 +37,35 @@ class EsConst(object):
     HITS = 'hits'
 
 
+class EsBulk(object):
+    OP_TYPE = '_op_type'
+    INDEX = 'index'
+    CREATE = 'create'
+    DELETE = 'delete'
+    UPDATE = 'update'
+
+
+class EsDocProperties(object):
+    INDEX = '_index'
```

Comment:
there is no convention for the underscore consts  - 

> SCROLL_ID = 'scroll_id'
    UNDERSCORE_SCROLL_ID = '_scroll_id'
 

vs

> class EsDocProperties(object):
    INDEX = '_index'
class EsBulk(object):
   INDEX = 'index'


I suggest _XXX for "_xxx" (_INDEX = "_index" )